website/docs/d/scc_posture_scopes.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scopes"
+description: |-
+  Get information about list_scopes
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scopes
+
+Review information of Security and Compliance Center posture scopes see.
```

Comment:
This sentence should be reworded as per the first comment provided. 

For more information, about security compliance scope, see [Managing scopes](https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-scopes)